system-helper/org.freedesktop.Flatpak.rules.in

```diff
@@ -1,11 +1,9 @@
 polkit.addRule(function(action, subject) {
-    if ((action.id == "org.freedesktop.Flatpak.app-install" ||
-         action.id == "org.freedesktop.Flatpak.runtime-install"||
+    if ((action.id == "org.freedesktop.Flatpak.runtime-install"||
          action.id == "org.freedesktop.Flatpak.app-uninstall" ||
          action.id == "org.freedesktop.Flatpak.runtime-uninstall" ||
          action.id == "org.freedesktop.Flatpak.modify-repo") &&
-        subject.active == true && subject.local == true &&
-        subject.isInGroup("@privileged_group@")) {
```

Comment:
I don’t think you want to remove the `privileged_group` check here. Otherwise actions like `app-uninstall` will be allowed for anyone (all users), as long as their session is local (i.e. not over SSH).

Reply:
Yes, makes sense. Thanks for pointing this out.
Addressed in the latest iteration of the patch.